### ISSUE TITLE
Add an antiflood for welcome

### ISSUE THREAD

**Issue by Flowiee** [GreyWolfDev/Executrix#29]
If there are massive joins in few times, Executrix should delete the previous welcome to not help the spam.

**Comment by Olgabrezel** [comment]
I don't think we can count group joins. And I'm not sure how much sense this makes after all... if the group is being spammed, you will be lucky to have each ID written in the chat so you can ban them. And editing the previous welcome to only say "$name $id joined" doesn't make sense either, because then Executrix will hit the API limits even faster than if she only sends one message per join...

**Comment by Flowiee** [comment]
Another idea:

- Delete the welcome and keep track of all new joins, then when the "massive" join is over send to the admin the list of users that joined, so they can check and see if they have to ban or something 

(Probably add a "ban all" button (?) or "close" if you just want to check who joined)
